src/Connections/FeedConnection.ts

```diff
@@ -95,10 +112,16 @@ export class FeedConnection extends BaseConnection implements IConnection {
                 url: this.feedUrl,
                 label: this.state.label,
             },
+            secrets: {
+                lastResults: [...this.lastResults.entriesDescending()].map(e => e[1]),
+            }
         }
     }
 
     private hasError = false;
+    private readonly lastResults = new QuickLRU<string, LastResultOk|LastResultFail>({ maxSize: 5 });
+    // Simply for fast lookup.
+    private lastResult: FeedError|true = true;
```

Comment:
I'd rather have this be a getter extracting things from lastResults, so that we don't have to set it twice.

Reply:
Done in [5fe3d4e](https://github.com/matrix-org/matrix-hookshot/pull/459/commits/5fe3d4e3d69d9ee361d4c7dec8e2f00a195e16c7)

Reply:
(Actually 9e61ff7d8919bec97f8a4f392b5f5fe288f6f0fd post-rebase, it seems)

Reply:
Looks alright!